ratelimiter/__init__.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Original work Copyright 2013 Arnaud Porterie
+# Modified work Copyright 2016 Frazer McLean
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+__author__ = 'Frazer McLean <frazer@frazermclean.co.uk>'
+__version__ = '1.2.0.post0'
+__license__ = 'Apache'
+__description__ = 'Simple python rate limiting object'
+
+# Async support is provided only on Python 3.5+
+if sys.version_info >= (3, 5):
+    from .async import AsyncRateLimiter as RateLimiter
```

Comment:
I think it would have been better to name the submodule `_async`.  This is valid because the Python devs made `async` a reserved keyword only in specific contexts (i.e. `async def`, `async with`, etc.) but this could be non-obvious to beginners or tools (see for example how the github parser marks this as a keyword).

Reply:
`AsyncRateLimiter` is a subclass of `RateLimiter`